py/visdom/__init__.py

```diff
@@ -42,6 +44,30 @@
 except ImportError:
     BS4_AVAILABLE = False
 
+try:
+    # TODO try to import https://github.com/CannyLab/tsne-cuda first? will be
+    # faster but requires more setup
+    import visdom.extra_deps.bhtsne.bhtsne as bhtsne
+
+    def do_tsne(X):
+        Y = bhtsne.run_bh_tsne(X, initial_dims=X.shape[1], verbose=True)
```

Comment:
@JackUrb - any thoughts on leaving `verbose` set to `True` so it's easier to see the status of the processing on the server?

Reply:
That's fine by me 👍